src/components/dnd-table-v2/dnd-table-row.tsx

```
@@ -4,7 +4,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-import { SxProps, TableCell, TableRowProps, Theme, Tooltip } from '@mui/material';
+import { SxProps, TableCell, TableRowProps, Theme } from '@mui/material';
+import { CustomTooltip } from '../tooltip/CustomTooltip';
```

ESLint

[error] `../tooltip/CustomTooltip` import should occur after import of `react-intl`

(import-x/order)